extendeddatacrossword.go

```diff
@@ -259,25 +256,6 @@ func (eds *ExtendedDataSquare) rebuildShares(
 		return nil, false, nil
 	}
 
-	if isExtendedPartIncomplete {
```

Comment:
[Question] I suppose there have not been any tests associated with this part of the code, as no test file has been updated in this PR, right?

Reply:
codecov claims these lines were covered: https://app.codecov.io/gh/celestiaorg/rsmt2d/blob/master/extendeddatacrossword.go

I assume via https://github.com/rootulp/rsmt2d/blob/1f1904acc114b41dff838ad1d530feacc1d9f199/extendeddatacrossword_test.go#L21